src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/AuthorizationProcessServiceImpl.scala

```diff
@@ -16,16 +16,16 @@ class AuthorizationProcessServiceImpl(authorizationProcessUrl: String, blockingE
   system: ActorSystem[_]
 ) extends AuthorizationProcessService {
 
-  val invoker: ApiInvoker = ApiInvoker(EnumsSerializers.all, blockingEc)(system.classicSystem)
-  val api: ClientApi      = ClientApi(authorizationProcessUrl)
-
+  val invoker: ApiInvoker      = ApiInvoker(EnumsSerializers.all, blockingEc)(system.classicSystem)
+  val clientApi: ClientApi     = ClientApi(authorizationProcessUrl)
```

Comment:
this rename is not useful right now and is creating a bit of noise, can we revert it?